packages/TorneloScoresheet/src/pages/GraphicalRecording/MoveOptionsSheet.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import Sheet from '../../components/Sheet/Sheet';
+import { colours } from '../../style/colour';
+import { ICON_CLOCK, ICON_HALF } from '../../style/images';
+import { PlayerColour } from '../../types/ChessGameInfo';
+import MoveOption from './MoveOption';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import MaterialIcon from 'react-native-vector-icons/MaterialIcons';
+import { useGraphicalRecordingState } from '../../context/AppModeStateContext';
+
+export type EditingMove = {
+  colour: PlayerColour;
+  moveIndex: number;
+};
+
+type MoveOptionsSheetProps = {
+  editingMove: EditingMove | undefined;
+  dismiss: () => void;
+};
+
+const MoveOptionsSheet = ({ editingMove, dismiss }: MoveOptionsSheetProps) => {
+  const recordingState = useGraphicalRecordingState();
+  const actions = recordingState?.[1];
+
+  const toggleDraw = actions?.toggleDraw;
+
+  const handleDrawOffer = () => {
+    console.log(editingMove);
+    if (!toggleDraw || !editingMove) return;
+    toggleDraw(
+      editingMove.moveIndex * 2 +
+        (editingMove.colour === PlayerColour.Black ? 1 : 0),
+    );
+    dismiss();
+  };
+
+  return (
+    <Sheet
+      visible={Boolean(editingMove)}
+      dismiss={dismiss}
+      title="Move options">
+      <MoveOption
```

Comment:
any reason we're not passing the function here? will it be part of the other ticket?

Reply:
The record time function? I think it's not implemented yet right? I thought that was your ticket

Reply:
yep no worries i will implement it